Earthfile

```diff
@@ -192,13 +209,24 @@ unit-test:
     # pkgname determines the package name (or names) that will be tested. The go
     # submodules must be specified explicitly or they will not be run, as
     # "./..." does not match submodules.
-    ARG pkgname = ./... github.com/earthly/earthly/ast/... github.com/earthly/earthly/util/deltautil/...
+    ARG pkgname = ./...
+
+    ARG setuppodman = true
     WITH DOCKER
-        RUN ./podman-setup.sh && \
-            if [ -n "$testname" ]; then testarg="-run $testname"; fi && \
-            go test -timeout 20m $testarg $pkgname
+        IF [ "$setuppodman" = "true" ]
+            RUN ./podman-setup.sh && \
```

Comment:
nit: to avoid duplicating the `go test ...` logic, maybe we could do something like:

```
RUN \
    if [ "setuppodman" = "true" ]; then ./podman-setup.sh; fi && \
    if [ -n "$testname" ]; then testarg="-run $testname"; fi && \
    go test -timeout 20m $testarg $pkgname
```

Reply:
Sounds good.  I was wondering, though, should this be using `-race`? I know it runs some integration tests, and `-race` is not really recommended with integration tests due to the goroutine limit...